usecases/backup/backend.go

```diff
@@ -572,3 +531,31 @@ func routinePoolSize(percentage int) int {
 	}
 	return 1
 }
+
+// RestoreClassDir restores classes on the filesystem directly from the temporary class backup stored on disk.
```

Comment:
comment(non-blocking): `RestoreClassDir returns a function that restores classes...`